actions/soak_helpers.go

```diff
@@ -5,57 +5,38 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
+	"time"
 
+	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/remotetestrunner"
 	"github.com/smartcontractkit/chainlink-testing-framework/config"
 	"github.com/smartcontractkit/chainlink-testing-framework/utils"
-	"github.com/smartcontractkit/helmenv/environment"
 )
 
-// BuildGoTestsDocker builds the go tests to run using docker, and returns a path to the test executable, along with
-// remote config options. This version usually takes longer to run, but eliminates issues with cross-compilation.
-//  Note: currentProjectRootPath and currentSoakTestRootPath are not interchangeable with utils.ProjectRoot and utils.SoakRoot
-//  when running in outside repositories. Keep an eye on when you need paths leading to this go package vs the current running project.
-func BuildGoTestsDocker(currentProjectRootPath, currentSoakTestRootPath, testsPath string) (string, error) {
```

Comment:
We'll still want these to help get around execution and compilation errors.

Reply:
We'll do that separately when moving soak tests.